app/Http/Controllers/Users/LDAPImportController.php

```diff
@@ -54,8 +75,9 @@ public function store(Request $request)
         if ($ldap_results['error']) {
             return redirect()->back()->withInput()->with('error', $ldap_results['error_message']);
         }
+
         return redirect()->route('ldap/user')
-            ->with('success', "LDAP Import successful.")
```

Comment:
Things like these little cleanups are really just fine, but when you're dealing with something this big, it's always best if you can keep those cleanups separated.

Reply:
I'll keep that in mind to keep the PR's cleaner.